test/built-ins/RegExp/prototype/flags/coercion.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2017 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-get-regexp.prototype.flags
+description: Boolean coercion of properties
+info: >
+  get RegExp.prototype.flags
+
+  [...]
+  4. Let global be ToBoolean(? Get(R, "global")).
+  6. Let ignoreCase be ToBoolean(? Get(R, "ignoreCase")).
+  8. Let multiline be ToBoolean(? Get(R, "multiline")).
+  10. Let dotAll be ToBoolean(? Get(R, "dotAll")).
+  12. Let unicode be ToBoolean(? Get(R, "unicode")).
+  14. Let sticky be ToBoolean(? Get(R, "sticky")).
```

Comment:
yes, I like to avoid loops and here we have the forEach on each one. It's better to just split them in different files and have a specific files for each case. If we find and errors or regression, the information will be more valuable and complete.